packages/theme/theme.ts

```diff
@@ -44,6 +114,37 @@ export const dark = {
   warning: '#FC540A',
   error: '#E86459',
   underline: 'rgba(229, 229, 229, 0.3)',
+  layout: colors.dark6,
+  btnPrimary: colors.primary5,
+  btnGoogle: colors.red2,
+  btnFont: colors.dark6,
+  btnFacebook: colors.primary9,
+  btnGray: colors.dark1,
+  btnWhite: colors.dark1,
+  text: colors.white1,
+  whiteText: colors.white1,
+  blueText: colors.primary5,
+  darkText: colors.white1,
+  grayText: colors.white1,
+  line: colors.white1,
+  darkLine: colors.gray5,
+  thickLine: colors.primary5,
+  mediumLine: colors.primary8,
+  border: '#EDEDED',
+  grayBorder0: colors.gray1,
+  grayBorder1: colors.gray2,
+  grayBorder2: colors.gray4,
+  grayBorder3: colors.gray5,
+  disableColor: colors.white2,
+  warnBackground: colors.red0,
+  warnColor: colors.red4,
+  tealBackGround: colors.green0,
+  tealColor: colors.green1,
+  labelInput: {
```

Comment:
Why does only this part have a different structure?
I think `labelInput.border` can be changed to `border` or `whiteBorder`
`labelInput.label` can be `whiteText`, background can be `darkBackground`

If you expect it'll make an error, you can add a comment about this with remain this code.

Reply:
I think labelInput is a well-formed structure, so I specified the default lableInput.

But looking again, I don't need to do this only for labelInput. Thanks for the comments